controllers/kyma_controller.go

```diff
@@ -20,30 +20,32 @@ import (
 	"context"
```

Comment:
Guess we don't need these import sorting changes in `kyma_controller`. Can you revert them?